printer_mods/hartk1213/Voron2.4_Pins_Mod/README.md

```diff
@@ -19,10 +18,19 @@ To complete this mod the following files will need to be reprinted
 - [a]_b_tensioner_pinned.stl
 
 **XY Joints**
+
+*NOTE: There are now 2 versions of these files , 1 set for Stock 2.4 and 1 set for  [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12)*
```

Comment:
Can you make this a relative link?

    [Arkeet's MGN12 mod](../../arkeet/mgn12)